### DISCUSSION TITLE
After editing comment, content is not saved

### DISCUSSION THREAD

**Discussion by EyalSeagull** [requarks/wiki#4742] [Error / Bug Report]
Hi, I've created a comment and saved it (so far so good). I've edited it and Wiki shows the updated content (so far so good). However, when I edit the comment **again** the **old content** is shown instead of the updated one.

The Wiki.js version is 2.5.219. Running on Ubuntu 20.04.3.

**Comment by ElectricDevDucK** [comment]
Hi, I also noticed that issue and can therefore confirm its existence. It seems the *edit function* for comments always bases the edit on the comments original version and not - as expected - on its latest version.

I am currently running a manually installed `Wiki.js 2.5.285` on `Linux - Debian (bullseye) 11 x64`.

**Comment by inthedark122** [comment]
Hello

Still have this issue

In the comment updating, we can see that only render property update: https://github.com/requarks/wiki/blob/v2.5.286/server/modules/comments/default/comment.js#L129

```
    await WIKI.models.comments.query().findById(id).patch({
      render: renderedContent
    })
```

Need also to update content

Could someone add PR for this issue?

**Reply by adroslice** [reply]
One word, one special character. :) https://github.com/requarks/wiki/pull/5646
